help-docs/marketplace/apps/ChainAegis.md

```diff
@@ -0,0 +1,33 @@
+**About this app:**
+
+ChainAegis is an AI-driven risk assessment and alerts platform. It provides rugpull alerts, security incident alerts, and you can check an address' reputation and risk types. ChainAegis also provides other security services such as smart contract auditing, on-chain analysis, asset tracking, and data APIs.
+
+**To get started:**
+
+1. Install the ChainAegis Miniapp from the [Collab.Land Marketplace](../getting-started)
+
+2. Your community members will get the latest 5 Rugpull and 5 security incidents through _/chainaegis alert_.
```

Comment:
rugpull and security incidents from where and across what chains?

Reply:
### Invert all directions to active steps:
e.g. Do X to achieve Y.

> 2. Your community members will get the latest 5 Rugpull and 5 security incidents through _/chainaegis alert_.

becomes
> 2. Use `/chainaegis alert` to pull the latest 5 Rugpull and 5 security incidents.

Reply:
OK, I'll modify all directions to active steps